modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/display/context/DDLViewRecordsDisplayContext.java

```diff
@@ -257,8 +257,7 @@ public String getDisplayStyle() {
 			navigationItem -> {
 				navigationItem.setActive(true);
 				navigationItem.setLabel(
-					HtmlUtil.extractText(
```

Comment:
@leticia-maciel, according to the [description](https://github.com/liferay/liferay-portal/blob/7cb912889719a27e3dff73a0501155c0f36610f9/portal-kernel/src/com/liferay/portal/kernel/util/HtmlUtil.java#L135-L148) of the method `extractText`, it should be removing the `script` tag. In light of this, can you please give more details why this is not working as intended?

Additionally, the solution suggested by you is changing a code not from DDL, actually, this is a code from Clay and this change may affect other Portlets. Therefore, I understand that we should reach Front-End Infrastructure team later to let them know about it.

Thanks.

Reply:
Hi @natocesarrego, the extractText method was indeed removing the script but it was the DDL name as well and this label was needed to show on navbar.

Reply:
Did you mean that the method `extractText` was removing the `script` tag and also the DDL name?

If so, do you think we can solve this issue only on DDL code replacing the method `extractText` by another one available on `HTMLUtil`?